### PR TITLE
FEC: add test for resize + losses

### PR DESCRIPTION
**Pull request by gavv**
Minor improvements in existing tests plus one new test for resize + losses.

Changes are splitted into commits for easy reviewing.

### CODE REVIEW

File: src/tests/roc_fec/target_openfec/test_writer_reader.cpp [hunk 1]
```diff
@@ -898,7 +898,7 @@ TEST(writer_reader, repaired_block_numbering) {
     }
 }
 
-TEST(writer_reader, resize_source_block_begin) {
+TEST(writer_reader, writer_block_sizes) {
```
Comment (dshil) [inline src/tests/roc_fec/target_openfec/test_writer_reader.cpp]: As we do not use reader we can remove decoder as well.

File: src/tests/roc_fec/target_openfec/test_writer_reader.cpp [hunk 2]
```diff
@@ -941,11 +983,14 @@ TEST(writer_reader, resize_source_block_begin) {
             CHECK(p);
             CHECK(p->fec());
             CHECK(p->fec()->source_block_length == block_sizes[n]);
+
+            check_audio_packet(p, rd_sn);
+            rd_sn++;
```
Comment (dshil) [inline src/tests/roc_fec/target_openfec/test_writer_reader.cpp]: It is worth to check if `wr_sn == rd_sn` at the end.

File: src/tests/roc_fec/target_openfec/test_writer_reader.cpp (hunk 1 above)
Comment (gavv) [reply src/tests/roc_fec/target_openfec/test_writer_reader.cpp]: Done.

File: src/tests/roc_fec/target_openfec/test_writer_reader.cpp (hunk 2 above)
Comment (gavv) [reply src/tests/roc_fec/target_openfec/test_writer_reader.cpp]: Indeed, done.

**Pull request merged**